aries-site/src/pages/foundation/wcag-checklist.mdx

```diff
@@ -0,0 +1,85 @@
+import { Checklist } from '../../components';
+
+There are three levels of compliance in the Web Content Accessibility Guidelines (WCAG) which reflect the priority of support:
+
+1. **A: Essential**
+   - If this is not met, assistive technologies will not be able to aid in reading, understanding or operating the page or product.
+2. **AA: Ideal Support**
+   - Required for multiple government and public websites. HPE strives for Level A and AA coverage on all products.
+3. **AAA: Specialized Support**
+   - Usually reserved for websites and applications that serve a specialized audience.
+
+### Success Criteria
+
+Each item on the checklist refers to a specific, testable success criterion within the WCAG 2.1 described by a reference number and title. Some accessibility issues align with multiple rules. In these cases, the most relevant criterion is identified for the checklist item.
+
+### Does this checklist guarantee a product is fully accessible?
+
+No. There is no such thing as a "100%" accessible product. However, this checklist prompts designers and developers to check for coverage of a wide range of disability conditions. Addressing these issues will improve experiences for all users who access your product and increase overall accessibility.
+
+## Global Code
```

Comment:
This entire page is serving as the checklist with a brief introduction and the main headers being the sections the checklist will be divided into. I wasn't sure about having another header on the page be "Checklist" if the entire page serves as the checklist. 

I think we will need a table of contents moving forward as it gets longer also since there will be a lot of sections.

Reply:
Mentioned this below, but it'd be nice to see the overall content outline for this page (sans detail). This will help with content hierarchy, headings and such.